src/EA.Weee.Domain/Organisation/Address.cs

```diff
@@ -134,6 +140,19 @@ private set
             }
         }
 
+        public string WebAddress
+        {
+            get => webAddress;
+            private set
+            {
+                if (value != null && value.Length > 50)
+                {
+                    throw new InvalidOperationException(string.Format(("Web address cannot be greater than 50 characters")));
```

Comment:
Maybe slightly picky but should it not be "Website address"

Reply:
@will-hope98  fixed.